modules/transactions/src/main/java/org/apache/ignite/internal/tx/impl/TxCleanupRequestHandler.java

```diff
@@ -50,7 +54,10 @@
  */
 public class TxCleanupRequestHandler {
     /** Tx messages factory. */
-    private static final TxMessagesFactory FACTORY = new TxMessagesFactory();
+    private static final TxMessagesFactory TX_MESSAGES_FACTORY = new TxMessagesFactory();
```

Comment:
Could we make it a constant and don't pass it as a parameter?

Reply:
I don't think it's worth it yet. It seems to me that it would be convenient to make an instance inside the factory, something like below, I’ll leave it as is for now.
```
public class TxMessagesFactory {
    puvlic static final TxMessagesFactory INSTANCE = new TxMessagesFactory();
}
```
But I was unable to reach an agreement with my colleagues.